src/_locales/en_US/messages.json

```diff
@@ -211,6 +211,30 @@
         "message": "Export user data",
         "description": ""
     },
+    "reset": {
+        "message": "Reset",
+        "description": ""
+    },
+    "reset_user_data": {
+        "message": "Reset browsing data",
+        "description": ""
+    },
+    "reset_data_confirm": {
+        "message": "This will delete all browsing data that Privacy Badger has collected and restore the extension to its default state. Some trackers which are currently blocked may be allowed through afterwards.",
+        "description": ""
+    },
+    "clear_all": {
+        "message": "Clear all",
+        "description": ""
+    },
+    "clear_all_data": {
+        "message": "Clear all tracker data",
+        "description": ""
+    },
+    "clear_data_confirm": {
+        "message": "This will delete everything that Privacy Badger knows about third-party trackers, including the pre-installed \"seed\" data. As a result, Privacy Badger will not block many common trackers until it learns about them again.",
```

Comment:
No need to mention "seed" data I think. "What the heck is it?", but also not important.

Reply:
"Many common trackers" is confusing to me. Some trackers will still be blocked?

Reply:
Remove "third-party". Too technical, not mentioned anywhere else.

Reply:
We just want to make it clear you know what you are doing by clicking this button. Maybe something like the following?

"This will delete everything Privacy Badger knows about trackers. As a result, Privacy Badger will not block anything until it has had a chance to re-learn from your browsing."